Description: Changed some of the makefile options.
 Added force to the link command.
 .
 ponysay (0.7-1) unstable; urgency=low
 .
   * Initial release
   * This is my first Debian package.
Author: Louis Taylor <kragniz@gmail.com>

--- ponysay-0.7.orig/Makefile
+++ ponysay-0.7/Makefile
@@ -2,4 +2,4 @@ install:
 	mkdir -p $(DESTDIR)/usr/share/ponies
 	cp -r ponies/*.pony $(DESTDIR)/usr/share/ponies/
 	install -Dm755 ponysay $(DESTDIR)/usr/bin/ponysay
-	ln -s ponysay $(DESTDIR)/usr/bin/ponythink
+	ln -sf ponysay $(DESTDIR)/usr/bin/ponythink
